slam_node:
  robot_model: agrob  # {agrob, rocha}
  world_frame_id: map_nn2
  use_semantic_features: False
  use_lidar_features: True
  use_image_features: False
  use_gps: False
  use_wheel_odometry: True

  camera_info:
    baseline: 0.12 # meters
    fx: 692.95849609375  # pixels
    cx: 692.95849609375  # pixels

  multilayer_mapping:
    # Grid map definitions
    grid_map:
      origin:
        x: -75.0
        y: -75.0
        z: -0.5
      width: 150.0
      lenght: 150.0
      height: 2.5
      resolution: 0.25 # meters
      save_map: True
      output_folder: "/home/andresaguiar/Desktop/"

  pf:
    n_particles: 500

    # Particles innovation
    sigma_xx: 0.5 # meters
    sigma_yy: 0.5 # meters
    sigma_zz: 0.3 # meters
    sigma_RR: 0.3 # radians
    sigma_PP: 0.3 # radians
    sigma_YY: 0.7 # radians
